### PR TITLE
[TD-2634] Block Conversation on dashboard for Zopim Users

### PR DESCRIPTION
**Pull request by pranavp99**
### What do you want to achieve?
-

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

-ran local script and checked if chat was being identified as zopim or not, further checked with dashboard code if chat is recognised as zopim chat or not

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/js/app/zendesk-chat-service.js [hunk 1]
```diff
@@ -242,6 +242,14 @@ var onTabClickedHandlerForZendeskConversations = function (event) {
     console.log("onTabClicked from zendesk: ", event, MCK_GROUP_MAP[event.tabId]);
     if (kommunicate._globals.zendeskChatSdkKey) {
         var currentGroupData = MCK_GROUP_MAP[event.tabId];
+        var conversationInfo = 
+        {
+          groupId: event.tabId,
+          metadata: {
+            "source" : "zopim"
+          }
+        }
```
Comment (gargshantnu) [inline webplugin/js/app/zendesk-chat-service.js:251]: formatting

File: webplugin/js/app/zendesk-chat-service.js [hunk 2]
```diff
@@ -242,6 +242,13 @@ var onTabClickedHandlerForZendeskConversations = function (event) {
     console.log("onTabClicked from zendesk: ", event, MCK_GROUP_MAP[event.tabId]);
     if (kommunicate._globals.zendeskChatSdkKey) {
         var currentGroupData = MCK_GROUP_MAP[event.tabId];
+        var conversationInfo = {
+          groupId: event.tabId,
+          metadata: {
+            "source" : "zopim"
+          }
+        }
+        Kommunicate.updateConversationMetadata(conversationInfo);
```
Comment (gargshantnu) [inline webplugin/js/app/zendesk-chat-service.js:251]: use some formatter extension in VS code, and before commit do a formatting.
global spacing of file is 4 spaces.

**Comment by gargshantnu** [comment]
also, have you not tested this code?
update PR description properly
Screenshot 2022-05-16 at 12 05 22 PM

**Review by gargshantnu — DISMISSED** [review]

**Comment by Prochnost55** [comment]
Why is this pr filed for master?

**Comment by pranavp99** [comment]
its filed for TD-2634

**Review by Prochnost55 — APPROVED** [review]

**Comment by Prochnost55** [comment]
@pranavp99 is it filed for TD-2634 or master?

**Review by Prochnost55 — CHANGES_REQUESTED** [review]
file the pr correctly
the base needs to be development

**Pull request closed without merging**